packages/threat-composer/src/customTypes/mitigations.ts

```diff
@@ -14,9 +14,15 @@
   limitations under the License.
  ******************************************************************************************************************** */
 import { z } from 'zod';
-import { ContentEntityBaseSchema, EntityLinkBaseSchema } from './entities';
+import { ContentEntityBaseSchema, EntityLinkBaseSchema, StatusSchema } from './entities';
+import { MITIGATION_STATUS_IDENTIFIED, MITIGATION_STATUS_IN_PROGRESS, MITIGATION_STATUS_RESOLVED, MITIGATION_STATUS_RESOLVED_ABANDONED, STATUS_NOT_SET } from '../configs';
```

Comment:
Upon reflection, 'Abandoned' carries a negative connotation - and that it is simply left behind or neglected. I propose we change to 'Will not action' which carries slightly less negative connotation in my opinion, and is clear that it's a _decision_:

`MITIGATION_STATUS_RESOLVED_WILLNOTACTION`